doc/placeholders.rst

```diff
@@ -3,9 +3,9 @@ Placeholders
 
 Once defined, a workflow is static unless you update it explicitly. But, you can pass
 input to workflow executions. You can have dynamic values
-that you use in the **parameters** fields of the steps in your workflow. For this,
+that you use in the **parameters** or **result_selector** fields of the steps in your workflow. For this,
 the AWS Step Functions Data Science SDK provides a way to define placeholders to pass around when you
-create your workflow. There are 2 mechanisms for passing dynamic values in a workflow.
+create your workflow. There are 3 mechanisms for passing dynamic values in a workflow.
```

Comment:
nice!
ideally these mechanisms have sub-headings and anchor links to them (is that possible? - not super familiar with .rst files)

Reply:
Agreed - adding sub-headings and anchor links will make the documentation more organized